blockchain/blockchain.go

```diff
@@ -39,6 +39,7 @@ func (e ErrIncompatibleBlock) Unwrap() error {
 	return e.Err
 }
 
+//go:generate mockgen -destination=../mocks/mock_blockchain.go -package=mocks github.com/NethermindEth/juno/blockchain Reader
```

Comment:
I assume `go generate` is enough to regenerate the mock right?

Reply:
YES

Reply:
Sorry, i forgot to mention, without `GOBIN` in your path and `mockgen` installed under `GOBIN` this won't work.